editor.planx.uk/src/@planx/components/Checklist/Public/hooks/useExclusiveOption.tsx

```diff
@@ -0,0 +1,23 @@
+import { FormikProps } from "formik";
+
+import { Option } from "../../../shared";
+
+export const useExclusiveOption = (
+  exclusiveOptions: Option[],
+  formik: FormikProps<{ checked: Array<string> }>
```

Comment:
nit: Seeing this prop many times in this PR points towards `useFormikContext()` being something worth considering. Not a blocker for sure.